cmake/init-compilation-flags.cmake

```diff
@@ -134,9 +135,10 @@ if(COMPILE_RUNTIME_LIGHT)
             "${PROJECT_BINARY_DIR}/tmp"
             "${PROJECT_BINARY_DIR}/check_coroutine_include.cpp"
             COMPILE_DEFINITIONS "${TRY_COMPILE_COMPILE_OPTIONS}"
+            CXX_STANDARD 20
```

Comment:
It's a strange place, I would move it here:
https://github.com/VKCOM/kphp/blob/32765c1ebdd1598236a7233f424b3f02de3522d6/cmake/init-compilation-flags.cmake#L21

Reply:
And at the same time, it is better to use the cmake style of specifying parameters for the property.

```cmake
set(CMAKE_CXX_STANDARD 20)
```

Reply:
it's not a cmake global parameter, it's for try_compile function. By default cmake do not pass c++ standart variable to COMPILE_DEFINITIONS, which is passed to try_compile function